templates/base.html

```diff
@@ -41,5 +41,22 @@
 
       <!--JavaScript at end of body for optimized loading-->
       <script type="text/javascript" src="static/js/materialize.min.js"></script>
+
+      <script type="text/javascript" src="static/js/justgage.js"></script>
+      <script type="text/javascript" src="static/js/raphael-2.1.4.min.js"></script>
+      <<script>
+
+
+
+var g = new JustGage({
+  id: "cpu_gauge",
+  min: 0,
+  max: 100,
+  title: "CPU Percent",
+  value: {{data["cpu_percent"]}},
```

Comment:
It might be advantageous to create a custom route for this data just because right now it grabs a whole lot of extra stuff that it doesn't need to grab.